citi-onboarding/pride-hack
# feature(apiTicket): ticket url database

## What I did
* Server updates
* Adjusted the `server/routes/index.js` to a new pattern
* Added a `/view/Ticket.js` and `/model/Ticket.js`

## How to test
1. Install all the necessary dependencies
2. Go to /server and run `nodemon` or `node .` on the terminal
3. To teste access `localhost:3001/admin/` and access the KeystoneJS
4. Go to section `Link Ingressos` and add a new URL
5. If the database works, you must see it at `localhost:3001/api/ticket
